meinberlin/assets/scss/utility.scss

```diff
@@ -27,6 +27,10 @@
     margin-top: $em-spacer;
 }
 
+.u-spacer-top-half {
+    margin-top: ($em-spacer * 0.5);
```

Comment:
Do we still use the em-spacer? Or should that be replaced by a rem.one?

Reply:
good point! I just used what is currently used to avoid messing around all spacings. @phillimorland what is your take on this?

Reply:
em-spacer is old and has not been replaced everywhere, going forward we should use spacer